src/Ledger/Foreign/HSLedger.agda

```diff
@@ -202,16 +201,24 @@ HSAbstractFunctions = record
   }
 instance _ = HSAbstractFunctions
 
-open TransactionStructure it
+open TransactionStructure it hiding (GovVote; GovProposal)
+open import Ledger.Gov.Properties it
 open import Ledger.Utxo it it
 open import Ledger.Utxo.Properties it it
 open import Ledger.Utxow.Properties it it
+open import Data.Rational
 
 instance
   _ = Convertible-Refl {⊤}
   _ = Convertible-Refl {ℕ}
   _ = Convertible-Refl {String}
 
+  Convertible-Rational : Convertible ℚ F.Rational
+  Convertible-Rational = λ where
+    .to (mkℚ n d _) → n F., d
+    .from (n F., zero) → 0ℚ -- TODO is there a safer way to do this?
```

Comment:
what's happening here?

Reply:
This is brand new stuff that I know nothing about.  According to git blame, it was contributed by @Soupstraw yesterday.

Reply:
oh, weird should it not be in this PR then?

Reply:
I think github shows this because `master` was merged into this branch instead of doing a rebase

Reply:
No, merging (instead of rebasing) does not show stuff that was recently merged into master as new on this branch.  I must have moved `Convertible-Rational` to a different place in the file and that's why it appeared as a new change, but I've since fixed that.